src/mudlet-mapper.xml

```diff
@@ -3186,58 +3769,131 @@ end
 
 if not area then mmp.echo(matches[3]..&quot; isn't a known area. Which one do you want to set?&quot;) return end
 
-local tmp = getRoomUserData(1, &quot;areaContinents&quot;)
```

Comment:
I see that data is not now being stored in roomId `1` at least here, has it moved to map or area userdata yet - or will it wait until we actually get to Release 3.0 (where we will update the ___default___ map format to a higher value that supports them)?